modules/crio_arch.adoc

```diff
@@ -0,0 +1,56 @@
+= Understanding CRI-O
+
+The link:http://cri-o.io[CRI-O] container engine provides a stable, more secure, and performant
+platform for running OCI (link:https://www.opencontainers.org/[Open Container Initiative])
+compatible runtimes.
+The CRI-O container engine can be used to launch containers and pods
+via OCI-compliant runtimes like runc, the default OCI runtime, or
+link:https://katacontainers.io/[Kata Containers].
+CRI-O's purpose is to be the container engine implementing
+the Kubernetes Container Runtime Interface (CRI) for {product-title} and Kubernetes,
+replacing the Docker service.
+
+CRI-O offers a streamlined container engine, while other container features
+are implemented as a separate set of innovative, independent commands. This
+approach allows container management features to develop at their own pace,
+without impeding the primary goal of CRI-O, and its only supported use case,
+as the container engine for Kubernetes-based installations.
+
+CRI-O's stability comes from the fact that it is developed,
+tested and released in tandem with Kubernetes major and minor releases and follows
+OCI standards.  For
+example, CRI-O 1.11 aligns with Kubernetes 1.11. The scope of CRI-O is tied to
+the link:https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md[Container Runtime Interface] (CRI).
+CRI took on the task of
+extracting and standardizing exactly what a Kubernetes service (kubelet) needed
+from its container engine. It did this to help stabilize Kubernetes container
```

Comment:
s/. It did this//

Reply:
Makes the sentence kind of long. Instead I did: s/It/The CRI team/